metron-deployment/roles/monit/templates/monit/monit.conf

```diff
@@ -0,0 +1,29 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+set logfile syslog
+set pidfile /var/run/monit.pid
+
+# check services at X second intervals
+set daemon 30
+include /etc/monit.d/*
+
+# allow http access
+set httpd port 2812 and
+    use address {{ ansible_host }}
```

Comment:
If you haven't already, I'd double check that the external hostname ends up in monit.conf. We could be advantaged by the fact that monit is listening everywhere regardless of what is in "use address".